code/model/DMSDocument.php

```diff
@@ -914,6 +923,13 @@ public function getCMSFields()
         $fields->add(new TextField('Title', 'Title'));
         $fields->add(new TextareaField('Description', 'Description'));
 
+        $downloadBehaviorSource = array(
+            '' => 'Default ('.Config::inst()->get('DMSDocument', 'default_download_behaviour').')',
```

Comment:
Sorry this hasn't been clear. I want each instance of an object to store how it should be downloaded - no object should store that it is "default behaviour" because if a CMS user wants to set a file to be explicitly downloaded and not opened in browser (and that's what the current default is) they won't make a change in the CMS. If then for some reason the default is chaned by a developer this file will now act differently if we don't store how it should be saved.

Think of it like storing the price of an item in a ecommerce platform. An order will store the price of the item as it was when ordered and not reference the existing price of an item. This is what we need to do.

Therefore, the DropdownField should use the config to populate the default selected item in the dropdown and not be used to store some kind of explicit "default" marker that causes behaviour changes when the code is changed.